content/en/users/cloud-compute/faq/_index.md

```diff
@@ -225,3 +225,28 @@ keys in the VMs to be able to access.
 If you don\'t have public IPs for all the VMs to be managed, you can also use
 one as a gateway as described in the
 [Ansible FAQ](https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-configure-a-jump-host-to-access-servers-that-i-have-no-direct-access-to).
+
+### How can I release resources without destroying my data?
+
+Whenever you delete a VM, the ephemeral disks associated with it will be also
+deleted so if you don't plan to use your VM for some time but still be able
+to recover the data or boot your VM in the same state, you will need to use
+some of the following strategies:
```

Comment:
(not proposing as suggestion so that you can format properly the line length)

```
> Whenever you delete a VM, the ephemeral disks associated with it will also be deleted.

If you don't plan to use your VM for some time but still want to be able
to recover the data or boot your VM in the same state, you will need to use
some of the following strategies:
```

Reply:
I rephrased a bit